x-pack/plugins/security_solution/public/detection_engine/rule_monitoring/components/execution_events_table/event_date_range_filter/event_date_range_filter.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiSuperDatePicker } from '@elastic/eui';
+import type { DateRange } from '../../../api';
+
+interface EventDateRangeFilterProps {
+  value: DateRange;
+  onChange: (value: DateRange) => void;
+}
+
+export function EventDateRangeFilter({ value, onChange }: EventDateRangeFilterProps): JSX.Element {
```

Comment:
Please use `React.memo` to avoid re-render when props do not change.

Reply:
I see. No I don't have any such examples of `EuiSuperDatePicker` with performance issues. I'm not wondering why not use the `EuiSuperDatePicker` component itself instead of creating a wrapper component around it and passing props to the wrapper component?

Reply:
I agree, it looks doubtful to incapsulate `EuiSuperDatePicker` in `EventDateRangeFilter` while we can use it directly especially taking into account there is no business logic inside. So far it looks simpler to use `EuiSuperDatePicker` directly. I made according changes.

Reply:
Thanks for the changes.